wled00/usermods_list.cpp

```diff
@@ -245,7 +245,8 @@ void registerUsermods()
   #endif
 
   #ifdef USERMOD_PIRSWITCH
-  usermods.add(new PIRsensorSwitch());
+  usermods.add(new PIRsensorSwitch("PIRsensorSwitch0"));
```

Comment:
If you want to use a string, use a PROGMEM string and keep original name.

Reply:
Noted